packages/block-library/src/gallery/style.scss

```diff
@@ -139,105 +139,107 @@
 
 .is-style-tiled {
 	.blocks-gallery-grid {
+		$grid-gap: 4px;
+
 		display: grid;
-		grid-gap: 4px;
+		grid-gap: $grid-gap;
```

Comment:
The `grid-gap` CSS property was renamed to `gap`. Also, it's not supported in IE11.

Reply:
Good note, thanks. Assuming this approach works out, IE11 support will likely be "fall back to default gallery style". Notably, IE11 doesn't support auto layout, which this feature relies on. There are [polyfills](https://github.com/FremyCompany/css-grid-polyfill/), but they're incomplete.